app/src/main/java/com/yterletskyi/happyfriend/features/settings/ui/SeparateLineItem.kt

```diff
@@ -0,0 +1,51 @@
+package com.yterletskyi.happyfriend.features.settings.ui
+
+import android.graphics.Canvas
+import android.graphics.Color
+import android.graphics.Paint
+import android.graphics.Rect
+import android.view.View
+import androidx.annotation.ColorInt
+import androidx.core.view.get
+import androidx.recyclerview.widget.RecyclerView
+import com.yterletskyi.happyfriend.common.x.dp
+
+class SeparateLineItem(@ColorInt private val lineColor: Int = Color.parseColor("#E8E8E8")) :
```

Comment:
can we please rename the class to be `SettingsLineItemDecoration` (like `SpaceItemDecoration` which we already have) ?